app/controllers/cloud_volume_controller.rb

```diff
@@ -569,6 +572,65 @@ def backup_restore_finished
     javascript_redirect :action => "show", :id => @volume.id
   end
 
+  def snapshot_new
+    assert_privileges("cloud_volume_snapshot_create")
+    @volume = find_by_id_filtered(CloudVolume, params[:id])
+    @in_a_form = true
+    drop_breadcrumb(
+      :name => _("Create Snapshot for %{model} \"%{name}\"") % {
+        :model => ui_lookup(:table => 'cloud_volume'),
```

Comment:
Ideally, the `:model` placeholder should not be used at all here. For the sake of nicer & easier translations, this should be just  `"Create Snapshot for Cloud Volume \"%{name}\""`. Same in other places in this PR.